components/MessageLink.vue

```diff
@@ -11,7 +11,7 @@ export default {
       default: ''
     },
 
-    suffic: {
+    suffix: {
```

Comment:
this change is quite wide ranging, but easily verified and each change is to a template